package.json

```diff
@@ -137,7 +138,7 @@
     "tslint": "^5.8.0",
     "tslint-config-swimlane": "^3.0.4",
     "tslint-loader": "^3.5.3",
-    "typescript": "^2.6.1",
+    "typescript": "^2.4.2",
```

Comment:
Should be ~2.4.2 if you want to keep it below 2.5.  ^2.4.2 will download 2.6.2 right now.

Reply:
thanks.  changing now.